lua/nvim-treesitter/ts_utils.lua

```diff
@@ -314,16 +320,17 @@ function M.is_in_node_range(node, line, col)
   end
 end
 
+--- @deprecated Use `vim.treesitter.get_node_range()` instead
 function M.get_node_range(node_or_range)
-  if type(node_or_range) == "table" then
-    return unpack(node_or_range)
-  else
-    return node_or_range:range()
-  end
+  vim.notify_once(
```

Comment:
this is exactly what the `vim.deprecate()` function was introduced for afaik

Reply:
I looked at that function too, but it seemed like it's for neovim core. I had to put the neovim version in the third parameter in which the function will be deleted (e.g. it will notify saying that the function will be deleted in neovim 0.9)